utils.py

```diff
@@ -173,13 +175,13 @@ def average_predictions(full_spectrogram, model):
     return tf.add_n(all_predictions)/len(segmented_spectro)
 
 
-def save_and_display_loss_through_epochs(epoch_range, loss, model_name):
+def save_and_display_loss_through_epochs(epoch_range, loss, model_name, y_label):
```

Comment:
You should change the name of the function `save_and_display_metric_through_epochs`

Reply:
Yes